registration-processor/pre-processor/registration-processor-packet-validator-stage/src/test/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessorTest.java

```diff
@@ -127,6 +128,9 @@ public class PacketValidateProcessorTest {
 	
 	@Mock
 	private NotificationUtility notificationUtility;
+
+	@Mock
+	DateTimeFormatter dateTimeFormatter;
```

Comment:
add 2 new unit test case for null and invalid date format handling
Also add a assert statement in the first positive test case to check if date is set correctly in the db object.

Reply:
assert statement in the first positive test case to check if date is set correctly in the db object. - this change is not found

Reply:
added a check to db object.